css/wsuwp-plugin-idonate-src.less

```diff
@@ -660,6 +660,15 @@
 	.ui-menu-item.ui-state-focus {
 		color: #981e32;
 	}
+	li.narrow-search {
```

Comment:
Instead of applying a custom style, you can use a CSS selector (`li:nth-child(11)`) to get the 11th child (which we know will be the message). If there are less than 11 results, it won't style them. 
This will eliminate some of the styling lag from jQuery, although it is bit more fragile if you change the number of results returned.